### ISSUE TITLE
[Feature] Support useTailwind

### ISSUE THREAD

**Issue by marklawlor** [marklawlor/tailwindcss-react-native#9]
There are instances where a style objects needs to be passed to a non-style attribute. 

```JSX
<MotiView
  from={{ opacity: 0 }}
  animate={{ opacity: 1 }}
  exit={{ opacity: 0 }}
/>
```

We should allow the use of `useParseTailwind` for these instances

```JSX
<MotiView
  from={useParseTailwind('opacity-0')}
  animate={useParseTailwind('opacity-100')}
  exit={useParseTailwind('opacity-0')}
/>
```

**Comment by marklawlor** [comment]
Prep work for this has been done - just need to update the `native-inline` visitor to add the inline styles.

**Comment by marklawlor** [comment]
Released as part of the Component API in `v0.1.1`
